contents/handbook/engineering/release-new-version.md

```diff
@@ -16,7 +20,7 @@ Hopefully we will not have to do many patch versions, but if between versions we
 
 > 💡 For the context of this guide `[version]` is interpreted as the version of the release (e.g. `1.29.0`).
 
-On the week before the release, on Thursday, we institute a code freeze (used to be Fridays but that led to a rush of PRs on the eleventh hour which made the release process trickier). We branch master into release-[version] and deploy that to our playground environment (playground.posthog.com) and ClickHouse test environment (samltest.posthog.net). Only bugfixes are allowed to be merged into this branch (and thus put on production) between the code freeze and the release going out. This gives us about three days to test if this release has any bugs.
+On the week before the release, on Wednesday, we institute a code freeze (used to be Fridays but that led to a rush of PRs on the eleventh hour which made the release process trickier). We branch master into release-[version] and deploy that to our playground environment (playground.posthog.com) and ClickHouse test environment (samltest.posthog.net). Only bugfixes are allowed to be merged into this branch (and thus into production) between the code freeze and the release going out. This gives us about three days to test if this release has any bugs.
```

Comment:
Extra Q: Do we still differentiate between `playground` and `samltest`? Both should be "ClickHouse test environments" now. CC @yakkomajuri

Reply:
Yeah we only do playground these days